pyccel/ast/low_level_tools.py

```diff
@@ -127,3 +128,32 @@ def object(self):
         """
         return self._obj
 
+#------------------------------------------------------------------------------
+class MacroUndef(PyccelAstNode):
+    """
+    A class for undefining a macro in a file.
+
+    A class for undefining a macro in a file.
+
+    Parameters
+    ----------
+    macro_name : str
+        The name of the macro.
+    """
+    _attribute_nodes = ()
+    __slots__ = ('_macro_name',)
+
+    def __init__(self, macro_name):
+        assert isinstance(macro_name, str)
+        self._macro_name = macro_name
+        super().__init__()
+
+    @property
+    def macro_name(self):
+        """
+        The name of the macro being defined.
```

Comment:
Shouldn't this be `The name of the macro to be undefined.`

Reply:
Yes absolutely. Copy paste error

Reply:
Fixed : fd12a6f02